docs/source/docker/getting_started/setup.rst

```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
 
 .. _docker-download-and-install:
 
-Download the Lightly Worker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Docker Installation
+^^^^^^^^^^^^^^^^^^^^
 
-The Lightly worker comes as a Docker image. Ask your account manager from Lightly for the credentials
-to download the image.
+Lightly Worker requires docker to run. We highly recommend a docker Installation 
+that supports using GPUs for hardware acceleration.
 
+**Check if docker is installed:**
 
-In short, installing the Docker image consists of the following steps:
-
-#. Copy the *container-credentials.json* to the instance you want to use for filtering 
-#. Authenticate Docker to download the Lightly image
-#. Pull the Docker image
-#. Check whether the container works
+.. code-block:: console
 
-**First**, we need to access the private container registry of Lightly. 
-You received a *container-credentials.json* file from your account manager.
+    sudo docker run --rm --gpus all nvidia/cuda:11.0.3-base-ubuntu20.04 nvidia-smi
 
-**Second**, to be able to download the Docker image you need to log in with these credentials. 
-The following command will authenticate your installed Docker account. 
-We assume *container-credentials.json* is in your current directory.
+You might get an error message like this if docker is installed but without GPU support
 
 .. code-block:: console
 
-    cat container-credentials.json | docker login -u _json_key --password-stdin https://eu.gcr.io
-
-If the above command does not work, try the following. And please make sure the 
-json format is correct (no sudden newlines etc.):
+    docker could not select device driver with capabilities gpu
 
-.. code-block:: console
 
-    cat container-credentials.json | docker login -u json_key --password-stdin https://eu.gcr.io
+If you don't have docker installed or without GPU support we recommend following
+our guide about :ref:`rst-docker-known-issues-faq-install-docker`. 
 
+.. note::
+   If you use a cloud instance (e.g. on AWS, GCP or Azure) Docker is most likely
```

Comment:
maybe make suggestion which "OS" should be selected when starting the machine (or do we have that info somewhere else?)

Reply:
Added at the beginning of the block that we recommend Linux